lib/lib-table-query/src/main/java/org/sagebionetworks/table/query/model/SynapseFunctionName.java

```diff
@@ -0,0 +1,23 @@
+package org.sagebionetworks.table.query.model;
+
+/**
+ * Supported Synapse Functions
+ */
+public enum SynapseFunctionName {
+    // User Id
+    CURRENT_USER(FunctionReturnType.LONG);
```

Comment:
You might want to actually use a USER_ID type, we could add it to this FunctionReturnType enum